dev-tools/ecs-migration.yml

```diff
@@ -1972,12 +1972,6 @@
 
 # renames to match Windows Event Log naming
 
-- from: type
-  to: winlog.api
-  alias: true
-  beat: winlogbeat
-  rename: false
-
```

Comment:
[Question]

Why is this not needed any more?

Reply:
We previously had `wineventlog` and `wineventlog-experimental` api's. The experimental API has been recently GA'd as `wineventlog-raw` in 8.18. For 9.0.0 we want to remove the old `wineventlog` api as it is less performant and the old way of rendering the events it is still achieved through config while using the new `wineventlog-raw` api.

So we will have a single api moving forward, that is why we can remove the winlog.api field references.

note that this changes are not going to be backported